operations/finance.md

```diff
@@ -241,7 +241,15 @@ Returns all accounting categories of the enterprise associated with the connecto
 * `Accommodation`
 * `FoodAndBeverage`
 * `Taxes`
-* ...
+* `Payments`
+* `ExternalRevenue`
+* `SundryIncome`
+* `Wellness`
+* `Sport`
+* `Technology`
+* `Facilities`
+* `Events`
+* `Tourism`
```

Comment:
I would say it's incorrect now. Previously, we had `...` meaning, that the list can be extended anytime. Now, we are missing it, it would mean, that we cannot extend it ever.